packages/fcl/src/app-utils/verify-signatures.js

```diff
@@ -55,7 +55,22 @@ const getVerifySignaturesScript = async (sig, opts) => {
       ? "verifyAccountProofSignatures"
       : "verifyUserSignatures"
 
-  const network = await config.first(["env", "flow.network"])
+  let network = await config.get("flow.network")
+  if (!network) {
+    network = await config.get("env")
+    if (network)
+      console.warn(
```

Comment:
We should use the `deprecate` utility from https://github.com/onflow/fcl-js/blob/master/packages/fcl/src/utils/deprecate.js

Reply:
My bad, I was just copying it from https://github.com/onflow/fcl-js/blob/25bf8242d62ee6289391691ba82a79da93760e9e/packages/sdk/src/build/build-get-events.js#L6
I'll throw in a PR for that one as well